test/bundler/expectBundled.ts

```diff
@@ -491,7 +491,7 @@ function expectBundled(
     outputPaths = (
       outputPaths
         ? outputPaths.map(file => path.join(root, file))
-        : entryPaths.map(file => path.join(outdir!, path.basename(file)))
+        : entryPaths.map(file => path.join(outdir || "", path.basename(file)))
```

Comment:
☝️ We now comply with Node's path rules regarding invalid input to `path.join()` so non-string values will cause a throw.

Reply:
nice